MaximStadnikov/task_tracker
# Add lesson 9 materials

### Summary

- Added "interactor", "sidekiq", "letter_opener" gems
- Move creating project to `Projects::Create` interactor
- Add sending notification when project created

### How it works

![image](https://user-images.githubusercontent.com/35406008/198876390-02ebccf5-da87-45ac-9f68-b940d157739d.png)
![image](https://user-images.githubusercontent.com/35406008/198876408-136c4b70-a5c0-4b49-a24a-6187dce0a0ce.png)

### Test plan

List of steps to manually test introduced functionality:

- Log in
- Go to http://localhost:3000
- Create new project
- Make sure notification has been sent

### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, services, scripts, etc.

### Checklist:
